Mathlib/GroupTheory/Abelianization.lean

```diff
@@ -307,3 +322,80 @@ instance [Finite (commutatorSet G)] :
   exact Finite.card_pos.ne'
 
 end commutatorRepresentatives
+
+/- # Subgroups containing commutator -/
+
+variable {G}
+
+/-- If G and H have multiplications *
+and φ : G → H is a surjective multiplicative map,
+and if G is commutative, then H is commutative.
+
+Since I only use it with groups,
+I should probably use function.surjective.comm_semigroup
+--/
+theorem Function.Surjective.comm
+    {G H : Type _} [Mul G] [Mul H] {φ : MulHom G H}
+    (is_surj : Function.Surjective φ)
+    (is_comm : Std.Commutative (· * · : G → G → G)) :
+    Std.Commutative (· * · : H → H → H) where
+  comm := fun a b ↦ by
+    obtain ⟨a', ha'⟩ := is_surj a
+    obtain ⟨b', hb'⟩ := is_surj b
+    simp only [← ha', ← hb', ← map_mul]
+    rw [is_comm.comm]
+
+theorem Subgroup.Normal.quotient_commutative_iff_commutator_le
+    {N : Subgroup G} (nN : N.Normal) :
+    Std.Commutative (· * · : G ⧸ N → _ → _) ↔ _root_.commutator G ≤ N := by
+  constructor
+  · intro hcomm
+    rw [commutator_eq_normalClosure]
+    rw [← Subgroup.normalClosure_subset_iff]
+    intro x hx
+    obtain ⟨p, q, rfl⟩ := hx
+    simp only [SetLike.mem_coe]
+    rw [← QuotientGroup.eq_one_iff]
+    rw [commutatorElement_def]
+    simp only [QuotientGroup.mk_mul, QuotientGroup.mk_inv]
+    rw [← commutatorElement_def]
+    rw [commutatorElement_eq_one_iff_mul_comm]
```

Comment:
```suggestion
        rw [SetLike.mem_coe, ← QuotientGroup.eq_one_iff, commutatorElement_def]
```

Reply:
I did not understand what you expected.

Reply:
I am just replacing
```lean
rw [foo]
rw [bar]
```
by
```lean
rw [foo, bar]
```
Not very important.

Reply:
I see. I had done it already, hence my misunderstanding.